src/plugins/intel_cpu/src/nodes/kernels/x64/jit_kernel_base.cpp

```diff
@@ -127,6 +144,45 @@ void JitKernelBase::uni_vpsubd(const Xbyak::Ymm& vDst,
     }
 }
 
+void JitKernelBase::uni_vsubpd(const Xbyak::Xmm& v_dst,
```

Comment:
Somewhere you use `vDst`, somewhere - `v_dst`. Can we align the variable names? Thanks

Reply:
Done